tests/src/core/DeviceLost.test.ts

```
@@ -36,15 +36,50 @@ describe("Device lost test", function () {
 
     engine.update();
 
-    // Force lost device.
-    engine.forceLoseDevice();
+    const resourceManager = engine.resourceManager;
+
+    // 注意：以下部分文件可能会版本迭代而解析报错，若出现此情况，只需更新对应版本的编辑器资产文件即可
+    const ambientLight = await resourceManager.load<AmbientLight>({
+      url: "https://mdn.alipayobjects.com/oasis_be/afts/file/A*t1inRbPh6VQAAAAAAAAAAAAADkp5AQ/ambient.json",
+      type: AssetType.Env
+    })
```

ESLint

[error] Insert `;`

(prettier/prettier)